hrui/hrui.php

```diff
@@ -934,3 +936,113 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+function _getMenuKeyMax($menuArray) {
+  $max = array(max(array_keys($menuArray)));
+  foreach($menuArray as $v) {
+    if (!empty($v['child'])) {
+      $max[] = _getMenuKeyMax($v['child']);
+    }
+  }
+  return max($max);
+}
+
+function civicrm_civicrm_navigationMenu(&$menu) {
+  // Create New Help Menu
+  _createHelpMenu($menu);
+
+  // Create New Developer Menu
+  _createDeveloperMenu($menu);
+}
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu
+ */
+function _createHelpMenu(&$menu) {
+  $maxKey = _getMenuKeyMax($menu);
+
+  $menu[$maxKey+1] = array(
```

Comment:
It looks like `_hrui_civix_insert_navigation_menu` will already find the max menu key. Can you try without using the `_getMenuKeyMax` and only passing in the values that aren't duplicated in `_hrui_civix_insert_navigation_menu`, like "permission" and "name"? I'm not 100% sure but it might work without it, and if it does then we can remove some code that we wouldn't need.